eq_translations/utils.py

```diff
@@ -109,5 +109,18 @@ def dumb_to_smart_quotes(string):
     return string.strip()
 
 
+APOSTROPHE = '\u0027'
+L_SQUOTE = '\u2018'
+R_SQUOTE = '\u2019'
+L_DQUOTE = '\u201C'
+R_DQUOTE = '\u201D'
```

Comment:
It would be clearer if character names were used e.g. `R_SQUOTE = '\N{RIGHT SINGLE QUOTATION MARK}'`.
https://www.cl.cam.ac.uk/~mgk25/ucs/quotes.html lists the relevant character names.